source/frontend/config-overrides.js

```diff
@@ -29,4 +29,33 @@ module.exports = {
     config.transformIgnorePatterns = ['node_modules/(?!(react-leaflet|@react-leaflet/core)/)'];
     return config;
   },
+
+  devServer: function (configFunction) {
```

Comment:
Huh, I don't see this called anywhere explicitly, is this function called automatically by some react-app-rewired process?

Reply:
Yeah, it's just part of the react dev server hosting